lib/api/core/models/repositories/userRepository.js

```diff
@@ -9,130 +11,134 @@ var
 /**
  * @class UserRepository
  * @extends Repository
- * @param {Kuzzle} kuzzle
- * @param opts
- * @constructor
  */
-function UserRepository (kuzzle, opts) {
-  Repository.apply(this, arguments);
-  this.collection = 'users';
-  this.ObjectConstructor = User;
-
-  if (opts !== undefined && opts.ttl !== undefined) {
-    this.ttl = opts.ttl;
+class UserRepository extends Repository {
+  /**
+   * @param {Kuzzle} kuzzle
+   * @param opts
+   * @constructor
+   */
+  constructor (kuzzle, opts) {
```

Comment:
niptpicking: 
```javascript
constructor (kuzzle, opts = {}) {
  super(kuzzle);

  this.collection = 'users';
  this.ObjectConstructor = User;

  if (opts.ttl !== undefined) {
    this.ttl = opts.ttl;
  }
}
```

anyway, I wonder if the assignment of `this.ttl` should be moved to superclass. (I mean: for now, it's not implemented in `RoleRepository` and `ProfileRepository` classes, but maybe it should ?)